src/API/OnboardingTasks.php

```diff
@@ -137,6 +148,37 @@ public function register_routes() {
 				'schema' => array( $this, 'get_public_item_schema' ),
 			)
 		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<id>[a-z0-9_-]+)/snooze',
+			array(
+				'args'   => array(
+					'task_id'      => array(
```

Comment:
This arg should be removed; we can use the `id` retrieved from the path.

Reply:
Whoops, that must have snuck in when I rebased 🤭